JavaErickHernandez/Day3/HashSets.java

```diff
@@ -0,0 +1,55 @@
+package bootcamp;
+import java.util.Scanner;
+import java.util.Set;
+import java.util.HashSet;
+
+public class HashSets {
+	static String numberOfInputs = "Enter the number of pairs: ";
+	static String tryAgainMessage = "Try Again";
+	static String enterFirstName = "Enter first name: ";
+	static String enterSecondName = "Enter second: ";
+	static String error = "Error!";
+	static HashSet<String> names = new HashSet<String>();
```

Comment:
When you variable is a Collection, it is good practice put name in singular and them the the type, example: tableList, nameSet, etc...